plugins/woocommerce/includes/class-woocommerce.php

```diff
@@ -221,6 +221,7 @@ private function init_hooks() {
 		add_action( 'init', array( 'WC_Emails', 'init_transactional_emails' ) );
 		add_action( 'init', array( $this, 'add_image_sizes' ) );
 		add_action( 'init', array( $this, 'load_rest_api' ) );
+		add_action( 'rest_api_init', array( 'WC_Site_Tracking', 'init' ) );
```

Comment:
I think we can hook into `init` instead of `rest_api_init` here, and get get rid of the call to `WC_Site_Tracking::init()` in

https://github.com/woocommerce/woocommerce/blob/6b158648e386646deba516937ea4b188fafa4443/plugins/woocommerce/includes/admin/class-wc-admin.php#L31

Reply:
Good call!  Updated in https://github.com/woocommerce/woocommerce/pull/37796/commits/6266eba02b3c379125a6fa5e1e643bc9b501423c.